app/actions.py

```diff
@@ -416,6 +438,76 @@ def cursorPageUp(self):
         self.cursorColDelta(self.penRow + penRowDelta), 0, 0)
     self.redo()
 
+  def cursorNeutralPageDown(self):
+    """
+    Moves the view and cursor down by a page or stops
+    at the bottom of the document if there is less than
+    a page left. Does not select any text and removes all
+    existing highlights.
+    Args:
```

Comment:
Is there a guideline these comments are following?
Just noticing that there have been blank lines above Args: line. I'm ok either way, just thought ya may want it consistent.

Reply:
Oh yeah my bad LOL this was how i saw it done on google's python style docs so I tried doing it. I'm still not used to it, but there's supposed to be a blank line before that.